posts/2024-10-03-introducing-grails-6-2-1.md

```diff
@@ -0,0 +1,134 @@
+
+## Bug Fixes and Improvements
+
+Grails 6.2.1 also addresses several bug fixes and improvements, including:
+
+* Fixes a lack of a “controller” attribute in GSP tags with a “resource” attribute
+* Fixes a bug in LinkGenerator when domain classes have a static id mapping to another property, that is not “id”
+* Fixes a problem finding the main class when running CLI commands
+
+These updates will enhance the stability and reliability of the framework.
+
+## Dependency Upgrades
+
+Keeping up with the latest dependencies is crucial for the health of any framework. In this release, we have upgraded various dependencies to their latest versions, including but not limited to the following:
+
+* `com.github.javaparser:javaparser-core` to version 3.25.10
+* `ch.qos.logback:logback-classic` to version 1.2.13
+* `org.springframework:spring-core` to version 5.3.39
+* `org.grails:grails-testing-support` to version 3.2.2
+* `org.apache.tomcat:tomcat-jdbc` to version 9.0.95
+* `org.grails.plugins:fields` to version 5.1.0
+* `org.grails.plugins:scaffolding` to version 5.1.2
+* `org.grails:grails-shell` to version 6.2.1
+* `org.grails:grails-wrapper` to version 4.0.0
+* `org.grails.plugins:gsp` to version 6.2.2
+* `org.grails.profiles:base` to version 6.0.0
+* `org.grails.profiles:web` to version 6.0.0
+* `org.grails.profiles:profile` to version 6.0.0
+* `org.grails.profiles:plugin` to version 6.0.0
+* `org.grails.profiles:web-plugin` to version 6.0.0
+* `org.grails.profiles:rest-api` to version 6.0.0
+* `org.grails.profiles:rest-api-plugin` to version 6.0.0
+* `org.grails.profiles:react` to version 6.0.0
+* `org.grails.profiles:vue` to version 6.0.0
+* `org.grails.profiles:angular` to version 9.0.0
+* And many more transitive dependencies.
+
+These upgrades ensure that Grails 6.2.1 remains compatible with the latest libraries and tools, providing developers with a robust and reliable environment. We recommend checking out the [GitHub release page](https://github.com/grails/grails-core/releases/tag/v6.2.1) for more information.
+
+## Installing Grails 6.2.1
+
+You can get started with Grails 6.2.1 today by visiting our [official website](https://start.grails.org/). Alternatively, you can quickly install Grails 6.2.1 using the [SDKMan](https://sdkman.io/).
+
+1. If you don't have SDKMan installed, follow the instructions at [SDKMan Installation Guide](https://sdkman.io/install/) to set it up.
+
+2. Once SDKMan is installed, open your terminal and run the following command to install Grails 6.2.1:
+
+    ````shell
+    sdk install grails 6.2.1
+    ````
+
+3. You're all set! To verify the installation, run:
+
+    ````shell
+    grails --version
+    ````
+
+## Upgrading Your Existing Applications to Grails 6.2.1
+
+If you already have a Grails application and want to upgrade to the latest version, follow these steps:
+
+1. Open the project in your favorite IDE (preferably JetBrain's IntelliJ IDEA).
```

Comment:
Just a small thing... Re: "JetBrain's IntelliJ IDEA", the company name is "JetBrains" so I think it probably makes sense to use "JetBrains' IntelliJ IDEA" or "JetBrains IntelliJ IDEA" but not "JetBrain's IntelliJ IDEA.

Reply:
This has been updated and also fixed on the release it was copied from.